docker/files/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM alpine:latest
+
+RUN mkdir -p /RackHD/downloads
+
+ADD https://bintray.com/artifact/download/rackhd/binary/ipxe/monorail-efi32-snponly.efi \
+    /RackHD/downloads/monorail-efi32-snponly.efi
+ADD https://bintray.com/artifact/download/rackhd/binary/ipxe/monorail-efi64-snponly.efi \
+    /RackHD/downloads/monorail-efi64-snponly.efi
+ADD https://bintray.com/artifact/download/rackhd/binary/ipxe/monorail.ipxe \
+    /RackHD/downloads/monorail.ipxe
+ADD https://bintray.com/artifact/download/rackhd/binary/ipxe/undionly.kpxe \
```

Comment:
Remove 'ADD https://bintray.com/artifact/download/rackhd/binary/ipxe/undionly.kpxe' and add 'https://bintray.com/artifact/download/rackhd/binary/syslinux/undionly.kkpxe', it's the latest update in master branch